docs/_documentations/mdt-vsc-firstproject.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: docs
-title: "Codewind for VS Code"
-description: "Codewind for VS Code"
+title: "Creating your first project"
+description: "Creating your first project"
```

Comment:
If a user comes to this page directly, such as through an internet search, they might not be aware of the table of contents and how this topic fits in with the bigger picture. Also, we might have a page for "Creating your first project" for each IDE, right?

If these are the cases, then I think we should mention the IDE as part of the title. Maybe something like "Creating your first Codewind project with VS Code" or "Creating your first VS Code Codewind project"?